src/hotspot/share/runtime/os.cpp

```diff
@@ -1723,6 +1723,9 @@ bool os::release_memory(char* addr, size_t bytes) {
   } else {
     res = pd_release_memory(addr, bytes);
   }
+  if (!res) {
+    log_info(os)("os::release_memory(" PTR_FORMAT ", " SIZE_FORMAT ") failed", p2i(addr), bytes);
```

Comment:
I think it's better to use `"os::release_memory failed (" PTR_FORMAT ", " SIZE_FORMAT ")"`. That way it's easy to match the error message in the test case with a simple substring test of `"os::release_memory failed"`. Otherwise it's hard to see that the regexp in the test ` "os::release_memory\\(0x[0-9a-fA-F]*,\\s[0-9]*\\)\\sfailed"` indeed would match the error message.

Reply:
Thanks for the suggestion.  Will make the check in test simple. (I have tested the pattern using online java pattern match and a local test program for it).

Thanks
Yumin